### PR TITLE
Add React testing library and Add Snapshot tests

### PR DESCRIPTION
**Pull request by JuanAlmeida12**
#### What is the purpose of this pull request?
As the title says.

#### Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: .vtexignore [hunk 1]
```diff
@@ -7,7 +7,8 @@ src/
 package.json
```
Comment (GustavoMA) [inline .vtexignore]: add `.prettierrc`

File: react/package.json [hunk 2]
```diff
@@ -1,10 +1,45 @@
...
-		"classnames": "^2.2.6",
-		"ramda": "^0.26.1",
-		"react-collapse": "^4.0.3",
-		"react-intl": "^2.7.2",
-		"react-motion": "^0.5.2"
-	}
-}
\ No newline at end of file
+  "scripts": {
+    "test": "jest",
```
Comment (GustavoMA) [inline react/package.json]:
```suggestion
    "pretest": "yarn",
    "test": "jest",
```

File: react/__tests__/__snapshots__/Footer.test.js.snap [hunk 3]
```diff
@@ -0,0 +1,398 @@
...
+                  />
+                </li>
+              </ul>
+            </div>
+          </div>
+        </div>
+      </section>
+    </footer>
+  </div>,
+  "debug": [Function],
```
Comment (rerissondaniel) [inline react/__tests__/__snapshots__/Footer.test.js.snap]: These functions are related to the test framework, I believe asFragment should be used to generate the snapshot.

File: react/__tests__/Footer.test.js [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+    }
+    return render(<Footer {...props} />)
+  }
+
+  it('should be rendered', () => {
+    expect(renderComponent()).toBeDefined()
+  })
+
+  it('should match the snapshot', () => {
+    expect(renderComponent()).toMatchSnapshot()
```
Comment (rerissondaniel) [inline react/__tests__/Footer.test.js]: `renderComponent()` returns an object that has functions that should be used in assertions. So, to snapshots the most adequate is `asFragment()`. Example [here](https://github.com/kentcdodds/react-testing-library#example)

**Pull request merged**